composer.json

```diff
@@ -11,13 +11,15 @@
         }
     ],
     "require": {
-        "php": "^7.3|^8.0",
+        "php": "^7.1|^7.2|^7.3",
         "ext-json": "*",
         "aws/aws-sdk-php": "^3.155",
-        "illuminate/support": "^8.52"
+        "illuminate/support": "5.4.*"
```

Comment:
why not using "^5.4.*" here? Just wanted to make sure this is compatible with 5.5, 5.6 etc ...

Reply:
Good spot sir!

Reply:
if it takes too much time that's something we can do when upgrading MIS (in a very very long time :-) )

Reply:
Should be all sorted now. 

As for the test matrix, I had to (or should I say couldn't figure out a better way to do it) make it really specific about which version of PHP and Testbench each Laravel version is tested against. 

If you think it should be tested against more PHP versions, that can be easily expanded on. 

I opted for using the lowest supported version of PHP for each Laravel version, unless it's lower than 7.1, then I used 7.1.